frontend/src/pages/main/Main.jsx

```diff
@@ -32,9 +32,9 @@ import moment from 'moment';
 
 const { mainText } = window.i18n;
 
-const NameWithInfo = ({ record, current, question }) => {
+const NameWithInfo = ({ record, title = null, current, question }) => {
   const { id, name, created_by, created, updated, updated_by, answer } = record;
-  let tmpName = name;
+  let tmpName = title.toString() || name;
```

Comment:
What if:
> undefined.toString();
Uncaught TypeError: Cannot read properties of undefined (reading 'toString')
> null.toString()
Uncaught TypeError: Cannot read properties of null (reading 'toString')

Also, are there any side effect for those who are still using the Datapoint name?

Reply:
I have check for those are using the Datapoint name, it look's fine.
- using ID
![image](https://github.com/akvo/wai-sdg-portal/assets/49269977/2b7c9c79-5734-4365-9a6e-2068fb03ff95)

- using Datapoint name
![image](https://github.com/akvo/wai-sdg-portal/assets/49269977/eb3eebd9-6277-4d12-9cc1-261b53d60b07)